gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
-distributionSha256Sum=7faa7198769f872826c8ef4f1450f839ec27f0b4d5d1e51bade63667cbccd205
```

Comment:
Unrelated to this PR and committed by accident, but without this change, I cannot import the project. Do you have the same @SerVB?

Reply:
Just checked. Turns out mine can't be imported both with and without this property. I have the following error logged:

```
Could not resolve all dependencies for configuration ':allopen-ide-plugin:compileClasspath'.
Multiple build operations failed.
    Unexpected state for resolution: Unknown
    Unexpected state for resolution: Unknown
    Unexpected state for resolution: Unknown
    Unexpected state for resolution: Unknown
Unexpected state for resolution: Unknown
```

However, it seems it doesn't affect me much since all the modules are imported properly and completion and navigation work.

Does it affect you? I see you've reverted the change, why?

Reply:
I don't want it to cause troubles when syncing with upstream - I'll keep this change only locally.